From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Mon, 16 May 2022 13:13:11 -0400
Subject: Skip failing tests

The rtpvp8 test is new for 1.20 but fails.
The rtpstorage test isn't new but started failing with 1.20

https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/1220
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/1221
---
 tests/check/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/check/meson.build b/tests/check/meson.build
index b44f1a0..5776067 100644
--- a/tests/check/meson.build
+++ b/tests/check/meson.build
@@ -90,7 +90,7 @@ if not get_option('rtp').disabled() and not get_option('rtpmanager').disabled()
     [ 'elements/rtph264' ],
     [ 'elements/rtph265' ],
     [ 'elements/rtpopus' ],
-    [ 'elements/rtpvp8' ],
+    [ 'elements/rtpvp8', true, ],
     [ 'elements/rtpvp9' ],
     [ 'elements/rtpbin' ],
     [ 'elements/rtpbin_buffer_list' ],
@@ -107,7 +107,7 @@ if not get_option('rtp').disabled() and not get_option('rtpmanager').disabled()
     [ 'elements/rtpptdemux' ],
     [ 'elements/rtprtx' ],
     [ 'elements/rtpsession' ],
-    [ 'elements/rtpstorage', false, [],  ['../../gst/rtp/gstrtpstorage.c',
+    [ 'elements/rtpstorage', true, [],  ['../../gst/rtp/gstrtpstorage.c',
 					'../../gst/rtp/gstrtpelement.c',
 					'../../gst/rtp/gstrtputils.c',
 					'../../gst/rtp/rtpstorage.c',
